typescript/packages/subsurface-viewer/src/layers/grid3d/fragment.fs.glsl.ts

```diff
@@ -5,29 +5,33 @@ precision highp float;
 
 in vec3 cameraPosition;
 in vec4 position_commonspace;
-in float property;
+flat in float property;
+in float property_interpolated;
 
 flat in vec3 normal;
 flat in int vertexIndex;
 
 uniform sampler2D colormap;
 
+uniform float valueRangeMin;
+uniform float valueRangeMax;
 uniform float colorMapRangeMin;
 uniform float colorMapRangeMax;
 
 uniform bool  isColoringDiscrete;
 uniform float colorMapSize;
+uniform highp int coloringMode;
```

Comment:
Does `highp` this really have any impact ? Just to store 0, 1, 2 and 4 ?

Reply:
Yes it has an impact. Without it the program does not work . I had to google it. The fragment shader needs this, The vertex shader do not. Even if it is the same uniform.

Reply:
ok